packages/core/src/sass/app/modules/reality-check.scss

```diff
@@ -133,17 +133,18 @@
         .radio-group--spending_limit {
             .dc-text {
                 font-weight: normal;
-                font-size: 12px;
-                line-height: 18px;
+                font-size: var(--text-size-xxs);
+                line-height: var(--text-lh-m);
             }
         }
     }
 }
 
 .dc-modal__container_reality-check {
-    max-width: 588px;
-    &.reality-check--summary {
-        max-width: 792px;
+    max-width: 588rem;
+
+    &.reality-check__summary {
+        max-width: 792rem;
```

Comment:
The values here need to be updated to `58.8rem` and `79.2rem`.
for more detail on `rem` you can refer to this link.
https://opensenselabs.com/blog/tech/rem-vs-em-vs-px-when-use-these-units

Reply:
@matin-binary just fixed this one